api/swagger.yml

```diff
@@ -32,9 +32,10 @@ info:
 
     ### Signatures
 
-    There are two endpoints that require signatures:
+    There are three endpoints that require signatures:
     * POST /account-creation-authorization: signed using an Ethereum key.
     * POST /transactions-pool: signed using BJJ key.
+    * PUT /transactions-pool: signed using BJJ key.
```

Comment:
A question: when the transaction data changes the transaction id changes too as it's based on transaction data hash. Am I correct?

Reply:
Yes, you are right. Fields, that take part in the transaction id creation: fromIdx, tokenID, Amount, Nonce, Fee

Reply:
Thanks for clarifying it.